index.html

```diff
@@ -212,6 +212,45 @@ <h6 class="text-muted">A website to generate a meaning for the abbreviation sele
             }
         });
     </script>
+
+    <!-- Script to save options as cookies -->
+    <script>
+            function setCookie(cname, cvalue, exdays) {
+                var d = new Date();
+                d.setTime(d.getTime() + (exdays*24*60*60*1000));
+                var expires = "expires="+ d.toUTCString();
+                document.cookie = cname + "=" + cvalue + ";" + expires + ";path=/";
+            }
+    
+            function getCookie(cname) {
+                var name = cname + "=";
+                var decodedCookie = decodeURIComponent(document.cookie);
+                var ca = decodedCookie.split(';');
+                for(var i = 0; i <ca.length; i++) {
+                    var c = ca[i];
+                    while (c.charAt(0) == ' ') {
+                    c = c.substring(1);
+                    }
+                    if (c.indexOf(name) == 0) {
+                    return c.substring(name.length, c.length);
+                    }
+                }
+                return "";
+            }
+    
+            function saveOptionAsCookie(el) {
+                setCookie(el.id, el.value, 90);
+            }
+            
+            function getOptionsFromCookies() {
+                var select_first = document.getElementById("first-select");
+                var select_second = document.getElementById("second-select");
+                select_first.value = getCookie("first-select");
+                select_second.value = getCookie("second-select");
+                run();
```

Comment:
Probably a silly question, but may I ask why this `run();` line is here?

Reply:
The `run();` line is to generate the Mystery Egg again after loading the cookied options.

Reply:
I noticed that the Randomize button is calling run() function, and I wanted to regenerate or randomize the mystery egg after loading the cookies.

Reply:
Ah, yep, got it.